### ISSUE TITLE
Feature request: Observe picture orientation if present in EXIF data

### ISSUE THREAD

**Issue by bdillahu** [mickael-kerjean/nuage#83]
Maybe I missed something, but rotating the picture to match orientation (if available) would be nice.

Just wanted to note it, not complaining :-)

**Comment by mickael-kerjean** [comment]
Hello there,

this feature has landed a few weeks ago (or was supposed to) and I can't seem to replicate this. If you don't have it working from your end, it's either a bug or a documentation issue, either way something needs to be fix.

Can you replicate the issue from the [demo](http://nuage.kerjean.me)?
Are you using the [transcoder](https://github.com/mickael-kerjean/nuage_transform)?

**Comment by bdillahu** [comment]
Hmm... works on the demo page, so it's my issue somewhere.

I'm not using the transcoder - haven't gotten into figuring out how that works with Docker stuff - looking at it that is probably my issue, isn't it?

**Comment by mickael-kerjean** [comment]
Yes it is. The idea by separating the transcoder from the rest was to avoid CPU intensive task to become the app bottleneck so that you don't have to worry about CPU consumption while browsing other stuff.

The docker stuff is simply here to avoid any dependency brain damage and making the installation a 3 commands job:
```
mkdir nuage && cd nuage
curl -X GET https://raw.githubusercontent.com/mickael-kerjean/nuage/master/docker/docker-compose.yml > docker-compose.yml
docker-compose up -d
```

Without docker, you need to run [this](https://github.com/mickael-kerjean/nuage_transform) process in parallel and glue everything together like this:
```
# run https://github.com/mickael-kerjean/nuage
TRANSCODER_URL=http://127.0.0.1:8335 node server/index.js

# run https://github.com/mickael-kerjean/nuage_transform
node src/index.js
```

In the long run, I will consider rewriting the backend in go to makes the install as convenient as possible but it's definitly not a priority for now.

**Comment by bdillahu** [comment]
Yep, fixed it to get the nuage_transform running... sorry for the extra time - I guess my only suggestion would be clarifying that both those pieces are needed...

And that is probably a case of me needing to read more closely, sorry.

**Comment by mickael-kerjean** [comment]
That's all good, it's very valuable to know the struggles someone can face during the install and it's hard to get the documentation aspect right without getting feedbacks

**Comment by mickael-kerjean** [comment]
I've update the documentation, if you encounter some issues, don't hesitate to ask for more help.
